src/methods/swap/transaction.ts

```diff
@@ -211,7 +211,7 @@ export const constructBuildTx = ({
             //  make sure we don't pass more with orders than API expects
             orders: params.orders.map((order) => {
               const sanitizedOrderData =
-                'makerAssetId' in order || 'takerAssetId' in order
+                'makerAssetId' in order
```

Comment:
what's wrong here though?

Reply:
yeah, that's what I didn't get. So first part of conditional  ( `makerAssetId' in order` ) makes it think it's some other type